### PR TITLE
Fix/earthquakes/schedule

### PR DESCRIPTION
**Pull request by J0N4S13**
## Description
Send a message to the earthquake discord in the last 10 minutes, if there was any earthquake felt in that time or if there was any earthquake with more than 3 magnitude.

## Task items:
- [ ] Add schedule code;
- [ ] Add a code in Earthquakes.js in services;

### CODE REVIEW

File: src/services/Earthquakes.js [hunk 1]
```diff
@@ -77,12 +87,61 @@ const getEarthquakes = async (client) => {
...
-      client.channels.get(channels.EARTHQUAKES_CHANNEL_ID).send(`***Sismos sentido dia ${yesterday.format('L')}:***\n${eventsSensed.join('\n')}`);
+      client.channels.get(channels.EARTHQUAKES_CHANNEL_ID).send(`***Sismos sentido dia ${yesterday}:***\n${eventsSensed.join('\n')}`);
+    }
+
+    // Send list of non-sensed earthquakes to Discord
+    if (events.length > 0) {
+      client.channels.get(channels.EARTHQUAKES_CHANNEL_ID).send(`***Sismos de ${yesterday}:***\n${events.join('\n')}`);
+    }
+  } catch (e) {
+    //
```
Comment (quetzyg) [inline src/services/Earthquakes.js:98]: What's the point of this catch?

File: src/services/Earthquakes.js [hunk 2]
```diff
@@ -77,12 +87,61 @@ const getEarthquakes = async (client) => {
...
+      obsRegion,
+    } = earthquake;
+    // Format time (hh:mm)
+    const formattedTime = moment(time).format('LT');
+
+    // Add earthquake to the correct list
+    if (sensed) {
+      // Add to list of sensed earthquake
+      eventsSensed.push(`${formattedTime}h - M${magType} **${magnitud}** em ${obsRegion} a ${depth}Km prof. **Sentido em ${local} com Int. ${degree}** ${shakemapref} // ${lat},${lon}`);
+    } else if (magnitud >= 3) {
```
Comment (quetzyg) [inline src/services/Earthquakes.js:130]: `magnitud` -> `magnitude`

File: src/services/Earthquakes.js [hunk 3]
```diff
@@ -33,6 +33,16 @@ const getByDate = async (date) => {
   }
 };
 
+const getRealTime = async (current) => {
+  try {
+    const earthquakes = await getAll();
+
+    return earthquakes.filter(({ time }) => ((current - new Date(time).getTime()) / 1000) <= 600);
+  } catch (e) {
```
Comment (quetzyg) [inline src/services/Earthquakes.js:41]: You might as well remove the whole try/catch, since this just throws what gets caught. Let it bubble up.

**Review by quetzyg — CHANGES_REQUESTED** [review]
Left some comments.

**Comment by tiagojpdias** [comment]
Closing due to inactivity.

**Pull request closed without merging**